docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>It is already here, cool. Maybe move it to the paragraph that introduces pipeline def?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>